app/AccountancyModule/Components/Participants/ParticipantList.php

```diff
@@ -238,10 +238,14 @@ public function createComponentFormMassParticipants() : BaseForm
             ->setDefaultValue('');
         $editCon->addSubmit('send', 'Upravit')
             ->setAttribute('class', 'btn btn-info btn-small')
-            ->onClick[] = fn(SubmitButton $button) => $this->massEditSubmitted($button);
```

Comment:
proč opouštíš `fn`?

Reply:
Momentálně to PHPStan bere jako chybu, protože ta metoda vrací void. Viz https://github.com/phpstan/phpstan/issues/3781

Reply:
Alternativou by bylo tu chybu ignorovat.